create_tag.py

```diff
@@ -119,12 +153,12 @@ def get_pullrequest_infos(args, repo, hashes):
     for i, commit_hash in enumerate(hashes):
         print(f"Fetching PR for commit {i+1}/{len(hashes)} ({commit_hash})")
         time.sleep(2)
-        pull_request = list_prs_for_hash(args, api, repo, commit_hash)
+        pull_request, author, reviewers = list_prs_for_hash(args, api, repo, commit_hash)
         if pull_request is not None:
             if repo == "cockpit-composer":
-                msg = f"- {pull_request.title} (#{pull_request.number})"
+                msg = f"- {pull_request.title} (#{pull_request.number}, Author: {author}, Reviewers: {reviewers})"
```

Comment:
nitpick: I propose to use `;` as the delimiter for PR number, author and reviewers, since multiple reviewers will be delimited using `,`.